src/mpid/ch4/src/ch4_types.h

```diff
@@ -311,6 +311,7 @@ typedef struct MPIDI_CH4_Global_t {
     OPA_int_t progress_count;
 
     MPID_Thread_mutex_t vci_lock;
+    MPIDI_vci_pool_t vci_pool;
```

Comment:
We may want to move workqueue inside the MPIDI_vci_t as well, so that we can support per vci workqueues.

Reply:
So, we are not getting rid of WorkQs? Wasn't in the room for the conclusion of the discussion.

Reply:
There was a discussion later where we decided to keep the WorkQs.

Reply:
Yes, I'll add that in this PR then.